foreman/data_refinery_foreman/surveyor/harmony.py

```diff
@@ -303,24 +477,27 @@ def create_variants(fields_list: List):
             space_variants.append(field.replace(" ", ""))
 
         for space_variant in space_variants:
-            variants.append(space_variant)
-            variants.append("characteristic [" + space_variant + "]")
-            variants.append("characteristic[" + space_variant + "]")
-            variants.append("characteristics [" + space_variant + "]")
-            variants.append("characteristics[" + space_variant + "]")
-            variants.append("comment [" + space_variant + "]")
-            variants.append("comment[" + space_variant + "]")
-            variants.append("comments [" + space_variant + "]")
-            variants.append("comments[" + space_variant + "]")
-            variants.append("factorvalue[" + space_variant + "]")
-            variants.append("factor value[" + space_variant + "]")
-            variants.append("factorvalue [" + space_variant + "]")
-            variants.append("factor value [" + space_variant + "]")
-            variants.append("sample_" + space_variant)
-            variants.append("sample_host" + space_variant)
-            variants.append("sample_sample_" + space_variant)  # Yes, seriously.
-
-    return variants
+            variants.add(space_variant)
+            variants.add("characteristic [" + space_variant + "]")
```

Comment:
It should be less expensive with
```suggestion
            variants.add(f"characteristic [{space_variant}]")
```

or join (here and below).

Reply:
I'm going to leave this as is and will revisit when I do array express.